list_docker_containers/tasks/main.yml

```diff
@@ -0,0 +1,18 @@
+- name: Check if docker is installed
+  command: which docker
+  register: which_docker
+  failed_when: no
+  changed_when: no
```

Comment:
Do the above two lines `failed_when: no` and `changed_when: no` make this command never fail and never count as changed?